Scripts/generate-test-config.sh

```diff
@@ -253,14 +252,26 @@ function resolve_credentials {
     # Ensure we don't output credentials
     [[ $LOG_LEVEL -ge $LOG_LEVEL_TRACE ]] && set +x
 
+    local circleci_execution_role
+    circleci_execution_role=$(aws ssm get-parameter --name "/mobile-sdk/${platform}/common/circleci_execution_role" 2> /dev/null || echo "")
+    readonly circleci_execution_role
+
+    if [ -z "${circleci_execution_role}" ]; then
+      die "Role not found for Circle CI in region: ${AWS_DEFAULT_REGION}"
+    fi
+
     local circleci_execution_role_arn
-    circleci_execution_role_arn=$(aws ssm get-parameter --name "/mobile-sdk/${platform}/common/circleci_execution_role" | jq -r .Parameter.Value)
+    circleci_execution_role_arn=$(echo "${circleci_execution_role}" | jq -r .Parameter.Value)
     readonly circleci_execution_role_arn
 
     local assume_role_creds
     assume_role_creds=$(aws sts assume-role --duration-seconds 3600 --role-arn "${circleci_execution_role_arn}" --role-session-name "IntegTest-$(date +%Y%m%d%H%M%S)")
     readonly assume_role_creds
 
+    if [ -z "${assume_role_creds}" ]; then
+      die "Failed to assume role credentials for Circle CI"
```

Comment:
nit: error clarification
```suggestion
      die "Failed to assume Circle CI execution role"
```